### PR TITLE
exit non-zero if any github tag(s) failed

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: codekit/cli/github_tag_version.py [hunk 1]
```diff
@@ -258,20 +271,34 @@ def main():
...
 
+    lp_fires = len(tag_exceptions)
+    if lp_fires:
+        eprint("ERROR: " + str(lp_fires) + " tag failures")
+
+        if args.debug:
+            for e in tag_exceptions:
+                eprint(str(e))
 
+        sys.exit(lp_fires if lp_fires < 256 else 255)
```
Comment (jonathansick) [inline codekit/cli/github_tag_version.py]: Add a couple spaces after the function for PEP8

File: codekit/cli/github_tag_version.py [hunk 2]
```diff
@@ -258,20 +271,34 @@ def main():
...
+                            raise
+
             else:
                 if args.debug:
                     print('No action for', repo.name,
                           'belonging to', team.name)
 
+    lp_fires = len(tag_exceptions)
+    if lp_fires:
+        eprint("ERROR: " + str(lp_fires) + " tag failures")
```
Comment (jonathansick) [inline codekit/cli/github_tag_version.py]: I'd prefer string formatting instead of concatenation here.

File: codekit/cli/github_tag_version.py [hunk 3]
```diff
@@ -29,6 +29,9 @@
 import urllib3
 from .. import codetools
 
+# https://stackoverflow.com/questions/5574702/how-to-print-to-stderr-in-python
+def eprint(*args, **kwargs):
```
Comment (jonathansick) [inline codekit/cli/github_tag_version.py]: Suggested function name: `print_exception`, and move to a common utilities module.

Comment (jhoblitt) [inline codekit/cli/github_tag_version.py]: This function prints to stderr, there isn't any special exception foo.  I've moved it into the codetools "util" namespace.

**Comment by jhoblitt** [comment]
@jonathansick Great comments -- I think I have addressed all points; take a second look?

**Comment by jhoblitt** [comment]
Rebased on master post merge of #25

**Pull request merged**